pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -750,6 +917,19 @@ func (r *ReconcileComplianceScan) deleteResultConfigMaps(instance *compv1alpha1.
 	return nil
 }
 
+func (r *ReconcileComplianceScan) deleteKubeletConfigConfigMaps(instance *compv1alpha1.ComplianceScan, logger logr.Logger) error {
```

Comment:
The delete logic here is relatively straight forward since each scan has it's own copy of the config maps for each node.

Related to the comments above, if we decide to use a single config map for each node and for each scan, we'll have to be careful how we clean them up, since we don't want to remove them if another scan is running.

The alternative would be to keep them around after the scan, and not clean them up, but that doesn't feel like a great solution since we're spreading configuration information around, increase the surface area for expose (or abuse).

In this case, we think erroring on the side of inefficiency by creating config maps for each scan + node combination is better than leaving config maps around or complicating the delete logic so a scan is aware of other scans.